sdk/search/azure-search-documents/CHANGELOG.md

```diff
@@ -18,6 +18,11 @@ and uplifted the properties to `SearchIndexerDataSourceConnection` and `SearchRe
 `NGramTokenFilter`, and `PathHierarchyTokenizer`.
 - Renamed `Similarity` to `SimilarityAlgorithm`.
 - Renamed `Suggester` to `SearchSuggester`.
+- Renamed fields `synonymMaps` to `synonymMapNames`, `analyzer` to `analyzerName`, 
+`searchAnalyzer` to `searchAnalyzerName` and `indexAnalyzer` to `indexAnalyzerName` 
+in `SearchField`, `SearchableField`.
+- Renamed `SimpleField` to `SimpleFieldBuilder`, `SearchableField` to `SearchableFieldBuilder`
+and `ComplexField` to `ComplexFieldBuilder`.
```

Comment:
If you were specific on the `listDataSourceConnection` in the previous CHANGELOG, you should probably fix that one up as well.